src/main/kotlin/no/nav/familie/klage/behandlingsstatistikk/PatchBehandlingsstatistikkController.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.klage.behandlingsstatistikk
+
+import no.nav.familie.klage.behandling.BehandlingRepository
+import no.nav.familie.kontrakter.felles.klage.BehandlingResultat
+import no.nav.familie.prosessering.domene.TaskRepository
+import no.nav.security.token.support.core.api.Unprotected
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/patch-statistikk")
+@Unprotected
+class PatchBehandlingsstatistikkController(
+    private val behandlingRepository: BehandlingRepository,
+    private val taskRepository: TaskRepository
+) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    @PostMapping("/")
+    fun patchTidligereBehandlingResultater() {
```

Comment:
Som Johan sa - legg inn en RequestBody liveRun her for å kunne teste dette før du kjører det. 
Og sjekk på liveRun.skalPersistere før du persisterer tasken :) 

Da kan du kjøre dryrun og liverun i preprod først - og dryrun i prod før du kjører liverun :)